src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -50,6 +51,29 @@
 public class JavadocDetailNodeParser {
 
     /**
+     * Options for errors listeners attached to ANTLR parser.
+     */
+    public enum ErrorListenerOption {
```

Comment:
@ps-sp This is a new option that was enabled in TC.
We aren't changing booleans for values in properties, but for internal use right now. You can see quite a few setters with this suppressed.